src/Foundation/Infrastructure/Cms/Settings/SettingsService.cs

```diff
@@ -192,6 +238,12 @@ public void UpdateSettings(Guid siteId, IContent content, bool isContentNotPubli
                     SiteSettings[siteId.ToString() + $"-{contentLanguage}"][contentType] = content;
```

Comment:
shouldn't` SiteSettings[$"{siteId}-default"][contentType] = content;` be set as well?